### PR TITLE
extract subtitles

### PR DESCRIPTION
**Pull request by lkischler**

### CODE REVIEW

File: Xabe.FFmpeg.Test/SubtitleTests.cs [hunk 1]
```diff
@@ -33,5 +32,34 @@ public class SubtitleTests
...
+                                                       .Start();
+
+            Assert.True(result.Success);
+            IMediaInfo resultInfo = await MediaInfo.Get(outputPath);
+            Assert.Equal(0, resultInfo.VideoStreams.Count());
+            Assert.Equal(0, resultInfo.AudioStreams.Count());
+            Assert.Equal(1, resultInfo.SubtitleStreams.Count());
+            Assert.Equal(expectedFormat, resultInfo.SubtitleStreams.First().Format);
+            if(checkOutputLanguage)
+            {
```
Comment (lkischler) [inline Xabe.FFmpeg.Test/SubtitleTests.cs]: should work after https://github.com/tomaszzmuda/Xabe.FFmpeg/issues/96

File: Xabe.FFmpeg/Model/IConversionResult.cs [hunk 2]
```diff
@@ -31,5 +31,10 @@ public interface IConversionResult
         ///     Conversion duration
         /// </summary>
         TimeSpan Duration { get; }
+
+        /// <summary>
+        ///     Arguments passed to ffmpeg
+        /// </summary>
+        string Arguments { get; }
```
Comment (lkischler) [inline Xabe.FFmpeg/Model/IConversionResult.cs]: Usefull for debbuging!

File: Xabe.FFmpeg/Streams/SubtitleStream.cs [hunk 3]
```diff
@@ -34,10 +34,28 @@ public string Build()
             var builder = new StringBuilder();
             builder.Append(_format);
             builder.Append(_split);
-            builder.Append(_language);
+            builder.Append(BuildLanguage());
             return builder.ToString();
         }
 
+        private string BuildLanguage()
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg/Streams/SubtitleStream.cs]: _language, language and Language could be confusing.

File: Xabe.FFmpeg/Streams/SubtitleStream.cs [hunk 4]
```diff
@@ -34,10 +34,28 @@ public string Build()
...
+        {
+            string language = string.Empty;
+            if(!string.IsNullOrEmpty(_language))
+            {
+                language = _language;
+            }
+            else if(!string.IsNullOrEmpty(Language))
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg/Streams/SubtitleStream.cs]: You could use else here

**Pull request merged**